plugin/kubernetes/setup.go

```diff
@@ -211,17 +210,6 @@ func ParseStanza(c *caddy.Controller) (*Kubernetes, error) {
 				continue
 			}
 			return nil, c.ArgErr()
-		case "resyncperiod":
```

Comment:
Instead of just removing, need to replace this case with a no-op... e.g.

```go
case "resyncperiod":
    continue
```

Following out deprecation policy, this will later be removed in a following version.